src/frontend/src/components/SearchBar/SearchBar.jsx

```diff
@@ -86,6 +79,9 @@ const useStyles = makeStyles((theme) => ({
     textTransform: 'capitalize',
     color: theme.palette.primary.main,
   },
+  searchResults: {
```

Comment:
This isn't being used that I can see.

Reply:
Yes its pointless, I was using it in a prior version. It can be removed.